sdk/personalizer/Azure.AI.Personalizer/src/Models/PersonalizerClient.cs

```diff
@@ -21,6 +21,8 @@ public class PersonalizerClient
         private readonly bool _isLocalInference;
         private string stringEndpoint;
         private string apiKey;
+        private float _interactionsSubsamplePercentage = 1.0f;
```

Comment:
Can you rename it to interactionsSubsampleRate and observationsSubsampleRate as you don't use percentage any more.

Reply:
renamed variables